targets/ingester/src/transform/agreements/index.ts

```diff
@@ -1,7 +1,10 @@
 import slugify from "@socialgouv/cdtn-slugify";
 import { SOURCES } from "@socialgouv/cdtn-sources";
+import { Question } from "@socialgouv/contributions-data-types";
+import { Agreement, IndexedAgreement } from "@socialgouv/kali-data-types";
```

Comment:
Prefer `import type`

```suggestion
import type { Question } from "@socialgouv/contributions-data-types";
import type { Agreement, IndexedAgreement } from "@socialgouv/kali-data-types";
```